src/runtime/runtime.ts

```diff
@@ -307,41 +261,15 @@ export class Runtime {
     return Object.isFrozen(recipe);
   }
 
-  // TODO(sjmiles): redundant vs. newArc, but has some impedance mismatch
-  // strategy is to merge first, unify second
-  async spawnArc({id, serialization, context, composer, storage, portFactories, inspectorFactory, storageManager}: SpawnArgs): Promise<Arc> {
-    const arcid = IdGenerator.newSession().newArcId(id);
-    const storageKey = new VolatileStorageKey(arcid, '');
-    const params = {
-      id: arcid,
-      fileName: './serialized.manifest',
-      serialization,
-      context: context || this.context,
-      storageKey,
-      slotComposer: composer,
-      pecFactories: [this.pecFactory, ...(portFactories || [])],
-      loader: this.loader,
-      inspectorFactory,
-      storageManager
-    };
-    return serialization ? Arc.deserialize(params) : new Arc(params);
-  }
-
   // static interface for the default runtime environment
 
   static async parse(content: string, options?): Promise<Manifest> {
-    return this.getRuntime().parse(content, options);
+    return staticRuntime.parse(content, options);
```

Comment:
will we get rid of these eventually too?

Reply:
Yes, ASAP.

Reply:
Somewhat related to this: I was digging into the internal checks failure, this seems to fix it: cl/345574453

I couldn't find other uses of Runtime.parse/Runtime.parseFile, but might not have checked all of the right places.